pkg/store/cache/factory.go

```diff
@@ -28,6 +28,9 @@ const (
 type IndexCacheConfig struct {
 	Type   IndexCacheProvider `yaml:"type"`
 	Config interface{}        `yaml:"config"`
+
+	// Available item types are Postings, Series and ExpandedPostings.
+	EnabledItems []string `yaml:"enabled_items"`
```

Comment:
We can probably create a dedicated string type for this, similar to IndexCacheProvider?